README.md

```diff
@@ -523,6 +523,7 @@ If you are using ProGuard you might need to add the following options:
 -keepclasseswithmembers class * {
     @com.squareup.moshi.* <methods>;
 }
+-keep @com.squareup.moshi.JsonQualifier class *
```

Comment:
`interface *` i think is just for interfaces? not that i know i'm talking about.